delivery_carrier_label_postlogistics/postlogistics/web_service.py

```diff
@@ -17,6 +17,12 @@
 _compile_itemnum = re.compile(r'[^0-9]')
 
 
+def _sanitize_string(string):
+    if not string:
+        return False
+    return string.replace("\u2018", "'").replace("\u2019", "'")
```

Comment:
Do they have a list of characters they support? I guess that's not the only ones they don't support?

Reply:
I asked for a list but I got only a vague answer about using utf-8 encoding.